x-pack/test/alerting_api_integration/security_and_spaces/group3/tests/maintenance_window/create_maintenance_window.ts

```diff
@@ -110,6 +110,71 @@ export default function createMaintenanceWindowTests({ getService }: FtrProvider
       });
     }
 
+    describe.skip('rRuleSchema validation', () => {
```

Comment:
is it a leftover?

Reply:
+1, please remove the `.skip` if this test is working